stories/Chip.stories.ts

```diff
@@ -68,6 +68,15 @@ ChipWithInfoBadge.args = {
   infoBadge: 'warning',
 };
 
+export const ChipWithRightBadge = Template.bind({});
+ChipWithRightBadge.args = {
+  label: 'With Right Badge',
+  rightBadge: {
+    label: 'novo',
+    type: 'negative',
+  },
+};
+
```

Comment:
Caso adicione o value no rightBadge

`
export const ChipWithRightBadge = Template.bind({});
ChipWithRightBadge.args = {
  label: 'Value in Right Badge',
  rightBadge: {
    label: '',
    type: 'negative',
    value: 100
  },
};
`